website/admin/views/admin_user_forms.py

```diff
@@ -0,0 +1,24 @@
+from flask_wtf import FlaskForm
+from wtforms import StringField, SubmitField, validators, SelectField, StringField, DecimalField
+from wtforms.validators import URL
+
+
+
+
+class AddNewRoomForm(FlaskForm):
```

Comment:
What's different from the forms in the internal blueprint? If they are the same, please re-use them by putting them in a shared file.